lib/connection/bind_uploader.js

```diff
@@ -58,7 +58,11 @@ function BindUploader(options, services, connectionConfig, requestId)
 		var rowNum = 0;
 		var fileCount = 0;
 		var strbuffer = "";
-		var tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), 'tmp')) + "\\";
+		var tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), 'tmp'));
+		if (tmpDir.indexOf('~') != -1) {
+			var tmpFolderName = tmpDir.substring(tmpDir.lastIndexOf('\\'));
+			tmpDir = process.env.USERPROFILE + '\\AppData\\Local\\Temp\\' + tmpFolderName;
```

Comment:
Do we \\AppData\\Local available on each supported platform? i.e. Windows, Linux, and Mac?

Reply:
only windows contain '~' on the path, should I add windows only?